rust/release/src/publisher.rs

```diff
@@ -171,46 +175,21 @@ impl Publisher {
                 registry,
                 repository,
                 tag,
-                ..
+                manifest,
             } => {
-                if let SourceArtifact {
-                    r#type: SourceType::Docker { .. },
+                if let SourceType::Docker {
+                    repository: source_repository,
                     ..
-                } = &self.source
+                } = &self.source.r#type
                 {
-                    let mut dockerfile = repo_workdir.clone();
-                    dockerfile.push("docker/actyx/Dockerfile");
-                    let target = format!("{}/{}:{}", registry, repository, tag);
-                    let args = [
-                        "buildx",
-                        "build",
-                        "--platform",
-                        "linux/arm64/v8,linux/amd64,linux/arm/v7,linux/arm/v6",
-                        "-f",
-                        &dockerfile.to_string_lossy(),
-                        &repo_workdir.to_string_lossy(),
-                    ];
-                    let cmd = Command::new("docker")
-                        .args(args)
-                        .output()
-                        .context(format!("building multiplatform docker images {:?}", args))?;
-
-                    anyhow::ensure!(
-                        cmd.status.success(),
-                        "Error running `docker {:?}` for {}\nstdout: {}\nstderr: {}",
-                        args,
-                        target,
-                        String::from_utf8(cmd.stdout)?,
-                        String::from_utf8(cmd.stderr)?
-                    );
-                    Ok(())
+                    docker_manifest_create(manifest, registry, source_repository, repository, tag)
```

Comment:
This function seems to call `manifest create`, replacing what previously calls `buildx build`.

When will `buildx build` be called?